histoqc/wsihandles/OpenSlideHandle.py

```diff
@@ -0,0 +1,93 @@
+from .WSIImageHandle import WSIImageHandle
+from histoqc._import_openslide import openslide
+from typing import Union
+class OpenSlideHandle(WSIImageHandle):
+    def __init__(self, fname):
+        print("OpenSlideHandle init")
+        self.fname = fname
+        self.osh = openslide.OpenSlide(fname)
+        self._has_bounding_box = True
+        self._bounding_box = self.__get_bounding_box()
+        
+        # get magnification factor from wsi slide
+        self._magnification_factor = self.osh.properties.get("openslide.objective-power") or \
+            self.osh.properties.get("aperio.AppMag")
+        
+        # get background color 
+        self._backfround_color = f"#{self.osh.properties.get(openslide.PROPERTY_NAME_BACKGROUND_COLOR, 'ffffff')}"
+
+    
+    def __get_bounding_box(self):
```

Comment:
Yes. Please see the following test code for the private functions.
```python
class Student:
    def _not_private(self):  # not private function
	    print('This is not private function.')
                 
    def __private(self): # private function
	    print('This is private function.')

std = Student()
# call no private function
std._not_private()

# Using name mangling to call private function
std._Student__private()

# AttributeError
std.__private() 
```
But some people said _Python's **private** functions are not actually private since user can use name mangling to call private function _
reference: https://www.geeksforgeeks.org/private-methods-in-python/

Reply:
super cool, i didn't know about this functionality, thanks!